sphinxcontrib/video.py

```diff
@@ -210,7 +219,10 @@ def visit_video_node_html(translator: HTMLTranslator, node: video_node) -> None:
 
 def depart_video_node_html(translator: HTMLTranslator, node: video_node) -> None:
     """Exit of the html video node."""
-    translator.body.append("</video>")
+    html_caption: str = f""
+    if node["caption"]:
+        html_caption += f"<p><span class=\"caption-text\">{node['caption']}</span></p>"
```

Comment:
can you use simple quotes in your fstrings to avoid escapes ? 

```suggestion
        html_caption += f'<p><span class="caption-text">{node["caption"]}</span></p>'
```